torch/_utils_internal.py

```diff
@@ -53,5 +53,5 @@ def get_source_lines_and_file(obj):
 
 
 TEST_MASTER_ADDR = '127.0.0.1'
-TEST_MASTER_PORT = 29500
+TEST_MASTER_PORT = 0
```

Comment:
Can we have some comment for this change? This is to enable dynamic ports, right? I assume there can only be one PG gang using port 0 at a time, even though the real port dynamically assigned by the system?

Reply:
Multiple PG gangs can use port 0 and TCPStore will bind to different ports for each one of them. The only caveat here is that you need to actually ask TCPStore which port was used and then ask the clients to connect to that port.